osmaxx_conversion_service/config/settings/common.py

```diff
@@ -252,6 +252,13 @@
         'PASSWORD': '',
         'DEFAULT_TIMEOUT': int(timedelta(days=1).total_seconds()),
     },
+    'high': {
+        'HOST': env.str('REDIS_HOST', default='localhost'),
+        'PORT': env.str('REDIS_PORT', default=6379),
+        'DB': 0,
+        'PASSWORD': '',
+        'DEFAULT_TIMEOUT': int(timedelta(days=1).total_seconds()),
+    },
```

Comment:
Consider eliminating the duplication here by pulling the identical dicts out to a local (private?) variable.